src/status_im/ethereum/ens.cljs

```diff
@@ -105,8 +105,12 @@
     :method "contenthash(bytes32)"
     :params [(namehash ens-name)]
     :on-success
-    (fn [hash]
-      (cb hash))}))
+    (fn [raw-hash]
+      ;; NOTE: it would be better if our abi-spec/decode was able to do that
+      (let [hash (subs raw-hash 130)
```

Comment:
does it work the same for ipfs and swarm?

Reply:
yes it's not decoding the contenthash itself at this point, just extracting it from the bytes soup returned by the contract